### ISSUE TITLE
conda env remove does not ask for confirmation before removing environment

### ISSUE THREAD

**Issue by justmarkham** [conda/conda#8509]
## Current Behavior

When I remove an environment using `conda env remove`, it does not ask for confirmation, but it does successfully remove the environment.

### Steps to Reproduce

My input:
```
conda env remove -n myenvironment
```

This is the entire output, which shows up immediately, and without asking for confirmation:
```
Remove all packages in environment /Users/kevin/miniconda3/envs/myenvironment:
```

## Expected Behavior

I expected it would ask for confirmation before removing the environment.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
     active environment : base
    active env location : /Users/kevin/miniconda3
            shell level : 4
       user config file : /Users/kevin/.condarc
 populated config files : /Users/kevin/.condarc
          conda version : 4.6.9
    conda-build version : not installed
         python version : 3.6.8.final.0
       base environment : /Users/kevin/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/kevin/miniconda3/pkgs
                          /Users/kevin/.conda/pkgs
       envs directories : /Users/kevin/miniconda3/envs
                          /Users/kevin/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.9 requests/2.18.4 CPython/3.6.8 Darwin/18.5.0 OSX/10.14.4
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /Users/kevin/.condarc <==
channels:
  - defaults
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /Users/kevin/miniconda3:
#
# Name                    Version                   Build  Channel
asn1crypto                0.22.0           py36hb705621_1    defaults
ca-certificates           2019.1.23                     0    defaults
certifi                   2019.3.9                 py36_0    defaults
cffi                      1.10.0           py36h880867e_1    defaults
chardet                   3.0.4            py36h96c241c_1    defaults
conda                     4.6.9                    py36_0    defaults
conda-env                 2.6.0                h36134e3_0    defaults
cryptography              2.6.1            py36ha12b0ac_0    defaults
idna                      2.6              py36h8628d0a_1    defaults
libcxx                    4.0.1                h579ed51_0    defaults
libcxxabi                 4.0.1                hebd6815_0    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                hd939716_3    defaults
ncurses                   6.1                  h0a44026_1    defaults
openssl                   1.1.1b               h1de35cc_1    defaults
pip                       9.0.1            py36hbd95645_3    defaults
pycosat                   0.6.3            py36h1de35cc_0    defaults
pycparser                 2.18             py36h724b2fc_1    defaults
pyopenssl                 17.2.0           py36h5d7bf08_0    defaults
pysocks                   1.6.7            py36hfa33cec_1    defaults
python                    3.6.8                haf84260_0    defaults
python.app                2                py36h54569d5_7    defaults
readline                  7.0                  h1de35cc_5    defaults
requests                  2.18.4           py36h4516966_1    defaults
ruamel_yaml               0.11.14          py36h9d7ade0_2    defaults
setuptools                36.5.0           py36h2134326_0    defaults
six                       1.11.0           py36h0e22d5e_1    defaults
sqlite                    3.27.2               ha441bb4_0    defaults
tk                        8.6.8                ha441bb4_0    defaults
urllib3                   1.22             py36h68b9469_0    defaults
wheel                     0.29.0           py36h3597b6d_1    defaults
xz                        5.2.4                h1de35cc_4    defaults
yaml                      0.1.7                hff548bb_1    defaults
zlib                      1.2.11               h60db283_1    defaults
```
</p></details>

**Comment by michaelaye** [comment]
Same here:

```bash

     active environment : py37
    active env location : /Users/klay6683/miniconda3/envs/py37
            shell level : 1
       user config file : /Users/klay6683/.condarc
 populated config files : /Users/klay6683/.condarc
                          /Users/klay6683/miniconda3/envs/py37/.condarc
          conda version : 4.6.11
    conda-build version : not installed
         python version : 3.7.3.final.0
       base environment : /Users/klay6683/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/klay6683/miniconda3/pkgs
                          /Users/klay6683/.conda/pkgs
       envs directories : /Users/klay6683/miniconda3/envs
                          /Users/klay6683/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.11 requests/2.21.0 CPython/3.7.3 Darwin/18.2.0 OSX/10.14.3
                UID:GID : 273771:2260
             netrc file : None
           offline mode : False

```

**Comment by jorisvandenbossche** [comment]
I also seem to remember that until recently, it *did* ask confirmation first.

**Comment by justmarkham** [comment]
For what it's worth, the alternative method for removing an environment is:

```
conda remove -n myenvironment --all
```

That method does ask for confirmation before proceeding.

**Comment by drorata** [comment]
I also have this issue and I second @jorisvandenbossche that I remember it used to ask for confirmation before removing environments.

**Comment by justmarkham** [comment]
Update: This issue still exists as of conda 4.8.0.

**Comment by drorata** [comment]
As well as in version 4.8.2.

**Comment by drorata** [comment]
Still present in version 4.8.4....

**Comment by cipri-tom** [comment]
I think the issue comes from here:

https://github.com/conda/conda/blob/1456484abee255344774eedd4edacd91bb8abc00/conda_env/cli/main_remove.py#L44-L47

`force` shouldn't be True.

**Comment by gwd666** [comment]
it does not even ask for confirmation during creation somehow "out of the blue"

**Comment by justmarkham** [comment]
I can confirm that this has now been fixed in my current conda version (24.7.1).

It appears that this was fixed as of 24.1.0 via #13440.
